src/ProjectOrigin.Electricity.Example/Program.cs

```diff
@@ -15,19 +15,46 @@ static async Task<int> Main(string[] args)
         switch (command)
```

Comment:
To manage more complex command line arguments, it may make sense to look at something like https://github.com/commandlineparser/commandline

Reply:
I totally agree, but i must say i am in a weird place about the example either more command-line like, or more test like with environment variables. Since it is NOT a full command-line client, it is more of a test client, so it could be that it should be rewritten as such.

I am just missing that testcontainers supported kind clusters ;)